VersionControlUI/src/items/VDiffComparisonPair.cpp

```diff
@@ -96,14 +117,23 @@ void VDiffComparisonPair::initializeForms()
 			->setColumnHorizontalAlignment(1, Visualization::LayoutStyle::Alignment::Left)
 			->setRowStretchFactor(0, 1)
 			->setRowStretchFactor(1, 1)
-			->put(0, 0, infoGrid->clone())
+			->put(0, 0, infoGrid)
```

Comment:
In the line below and the `addForm` line you are unnecessarily cloning and thereby leaking memory. Don't clone anything, since you overwrite the `infoGrid` and `diffGrid` and `container` variables with new objects anyway later.